src/layers/h3-hexagon-layer/h3-hexagon-layer.js

```diff
@@ -224,7 +224,7 @@ export default class HexagonIdLayer extends Layer {
 
     allData.forEach((d, index) => {
       const id = getHexId(d);
-      if (typeof id !== 'string' || !id.length) {
+      if (!h3IsValid(id)) {
```

Comment:
This should also remove the need for the `id &&` checks on L233-L234 below.

Reply:
Ok, also updated